storage/bucket.go

```diff
@@ -750,6 +750,7 @@ func (ua *BucketAttrsToUpdate) toRawBucket() *raw.Bucket {
 	}
 	if ua.Lifecycle != nil {
 		rb.Lifecycle = toRawLifecycle(*ua.Lifecycle)
+		rb.ForceSendFields = append(rb.ForceSendFields, "lifecycle")
```

Comment:
I think this should be "Lifecycle". Maybe it does not matter though. Under the hood I think it uses reflection with field names though to force send them.

Reply:
https://github.com/googleapis/google-api-go-client/blob/master/internal/gensupport/json.go#L187

Reply:
Interesting-- in the actual request body both lifecycle and rule are lower case. Can change if you think it makes more sense though.

Reply:
Correct, the json will have lower case because of struct tags: https://pkg.go.dev/google.golang.org/api/storage/v1?tab=doc#Bucket

Reply:
Switched to `Lifecycle`.